scripts/google-free.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+sed -i -e '$aGOOGLE_FREE=1' .env.release
+
+#------------------------------------
+# Remove Android Remote Notifications
+#------------------------------------
+
+yarn remove react-native-notifications
+rm android/app/google-services.json
+sed -i -e '/com.wix.reactnativenotifications.RNNotificationsPackage;/d' android/app/src/main/java/im/status/ethereum/MainApplication.java
+sed -i -e '/com.google.gms:google-services/d' android/build.gradle
+sed -i -e '/react-native-notifications/,+2d' android/settings.gradle
+FIREBASE_LINE=`grep -n 'com.google.firebase.messaging.default_notification_icon' android/app/src/main/AndroidManifest.xml | cut -d : -f 1`
+sed -i -e "$((FIREBASE_LINE-1)),+5d" android/app/src/main/AndroidManifest.xml
+sed -i -e 's/\["react-native-notifications" :refer (Notifications)\]//' src/status_im/notifications/android_remote.cljs
```

Comment:
Is this really the only way to do this? This is not very petty, I have to tell you.

Reply:
This stiff will EASILY break in the future, it's very fragile.

Reply:
Hi jakubgs, Thank you very much for you review.
> Is this really the only way to do this? This is not very petty, I have to tell you.

We can't condition some things like java imports, removing them at the build time seems only option 🤔

> This stiff will EASILY break in the future, it's very fragile.

I improved the script and removed breakable parts 👍. The only part which now can break(IMO) is 
`sed -i -e 's/\["react-native-notifications" :refer (Notifications)\]//' ...`
Sorry, I don't know how I can replace this one (looking for suggestion)

Reply:
But why are you using Sed on `android/build.gradle`? That can be conditional.

>We can't condition some things like java imports, removing them at the build time seems only option

Why can't we use [`Class.forName()`](https://docs.oracle.com/javase/7/docs/api/java/lang/Class.html#forName(java.lang.String))? That can be conditional.

Reply:
I'm not a Clojure wizard, but what about the `:provided` option?
https://stackoverflow.com/a/34134158

Maybe @flexsurfer has some better ideas.

Reply:
@jakubgs, all sed commands are removed, Please let me know if anything else can be improved.
Thanks, @flexsurfer for pointing me to [Conditional Reading](https://shadow-cljs.github.io/docs/UsersGuide.html#_conditional_reading), great feature.

Reply:
That's great, thank you!